eventstore_client.go

```diff
@@ -81,7 +81,10 @@ func get(path string) <-chan getResult {
 	go func() {
 		log.WithField("url", url).Debug("GET request to EventStore")
 
-		response, err := client.Get(url)
+		client := &http.Client{}
```

Comment:
Is there a reason why the client initialization can't be done in the `initializeClient` function? This code here does not respect timeouts.

Reply:
Yes you're right,  I did see the initializeClient function.
I've just removed that line.